src/interface/src/app/plan/scenario-results/scenario-results.component.ts

```diff
@@ -1,27 +1,29 @@
 import { Component, Input, OnChanges, SimpleChanges } from '@angular/core';
 import { ProjectAreaReport } from '../project-areas/project-areas.component';
-import { generateDummyData } from './temp-helper';
-import { ScenarioResult } from '../../types';
+import { Scenario, ScenarioResult } from '../../types';
 
 @Component({
   selector: 'app-scenario-results',
   templateUrl: './scenario-results.component.html',
   styleUrls: ['./scenario-results.component.scss'],
 })
 export class ScenarioResultsComponent implements OnChanges {
-  @Input() results: ScenarioResult | null = null;
+  @Input() scenario: Scenario | null = null;
```

Comment:
do we still need to pass in all `scenario` to this component, instead of just the results?

Reply:
Good point! Passing just results now